apps/system/style/chrome/chrome.css

```diff
@@ -246,6 +246,7 @@
   visibility: visible;
 }
 
+.appWindow.attentionWindow.active .chrome,
```

Comment:
Hi Yura,
Do you think you can have someone from system look at this change?  I'm not quite sure what else it can impact besides the callscreen.
Thanks,
-tamara